doorway-ui-components/src/global/tokens/borders.scss

```diff
@@ -16,4 +16,5 @@
   --bloom-shadow-md: 0 4px 6px -1px rgb(0 0 0 / 0.1), 0 2px 4px -2px rgb(0 0 0 / 0.1);
 
   --doorway-standard-border: var(--bloom-border-1) solid var(--bloom-color-gray-450);
+  --doorway-border-100: 100px;
```

Comment:
I believe this is 30 px in Figma, no?  Also looks like `--bloom-rounded-full` would get us close enough to the same thing.

Reply:
It seems anything over ~15px works with the same affect. Should I just set it to 15?

Reply:
I set this to an existing var that works.